src/Controller/Plugin/Forward.php

```diff
@@ -193,7 +193,7 @@ protected function detachProblemListeners(SharedEvents $sharedEvents)
 
                     foreach ($classArray as $class) {
                         if ($currentCallback instanceof $class) {
-                            $sharedEvents->detach($id, $currentEvent);
+                            $sharedEvents->detach($currentEvent, $id);
```

Comment:
This is a super-subtle bc break. Is it possible to revert this in the EVM instead?